docs/intro/install.rst

```diff
@@ -118,6 +118,27 @@ Once you've installed `Anaconda`_ or `Miniconda`_, install Scrapy with::
 
   conda install -c conda-forge scrapy
 
+To install Scrapy on Windows using ``pip`` :-
+
+1. Install latest version of `Python`_ (Recommended to tick [Add Python to PATH] checkbox during installation)
+
+2. Install or upgrade pip::
```

Comment:
This step should not be needed according to https://pip.pypa.io/en/stable/installing/#do-i-need-to-install-pip